orix/quaternion/orientation.py

```diff
@@ -361,16 +402,30 @@ def symmetry(self):
         """Symmetry."""
         return self._symmetry[1]
 
+    @symmetry.setter
+    def symmetry(self, value):
+        if not isinstance(value, Symmetry):
+            raise TypeError(
+                "Value must be an instance of orix.quaternion.symmetry.Symmetry."
```

Comment:
The `Symmetry` class is reachable from the `orix.quaternion` module, and should be imported from here.

```suggestion
                "Value must be an instance of orix.quaternion.Symmetry."
```